src/views/Compose.vue

```diff
@@ -387,6 +387,11 @@ export default {
             and avert rescheduling. The percentage should be set to the lowest value for which the amount
             of data is acceptable to meet the science goal of the request.`
           },
+          configuration_repeats: {
+            hide: simpleInterface,
+            desc: `The number of times this Request's set of Configurations should be repeated. This is useful for
+            nodding back and forth between a set of Targets that are too far apart to use Dithering.`
```

Comment:
Super nitpick - seems like in the other tooltips request/configuration/target isn't capitalized. I can go either way, but maybe keep it lowercase for this?

Reply:
I think it's appropriate to capitalize here. I distinguish it by if its refering to the structure of the model then capitalize it, but if its refering to the generic "observation request" then leave it lowercase.